src/Appwrite/Platform/Modules/Databases/Http/Databases/Collections/Documents/Upsert.php

```
@@ -95,7 +95,7 @@ public function __construct()
             ->callback($this->action(...));
     }
 
-    public function action(string $databaseId, string $collectionId, string $documentId, string|array $data, ?array $permissions, ?string $transactionId, ?\DateTime $requestTimestamp, UtopiaResponse $response, Document $user, Database $dbForProject, Event $queueForEvents, Context $usage, TransactionState $transactionState, array $plan, Authorization $authorization): void
+    public function action(string $databaseId, string $collectionId, string $documentId, string|array $data, ?array $permissions, ?string $transactionId, ?\DateTime $requestTimestamp, UtopiaResponse $response, User $user, Database $dbForProject, Event $queueForEvents, Context $usage, TransactionState $transactionState, array $plan, Authorization $authorization): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$requestTimestamp'. (undefined)

(UnusedFormalParameter)